services/credential-server/src/modules/signify/signifyApi.ts

```diff
@@ -74,7 +74,7 @@ export class SignifyApi {
 
   async resolveOobi(url: string): Promise<any> {
     const alias = new URL(url).searchParams.get("name") ?? uuidv4();
```

Comment:
What happend if not `alias`?

Maybe: 
`if (!alias) throw new Error("Alias is required but not provided.");`

Reply:
This is for the test server so doesnt matter